### ISSUE TITLE
default flyout menu layer 3 does not appear to be clickable

### ISSUE THREAD

**Issue by AustinHaws-Utah** [utahdts/utah-design-system#236]
From Ty:
default flyout menu layer 3 does not appear to be clickable, inline and mega works great but not "flyout"

code:
```html
<html>
  <head>
    <link
      rel="stylesheet"
      href="https://unpkg.com/@utahdts/utah-design-system-header/dist/style.css"
    />

    <script src="https://unpkg.com/@utahdts/utah-design-system-header/dist/utah-design-system-header.umd.js"></script>
    <script>
      //wait for the utah header to be ready
      document.addEventListener("utahHeaderLoaded", () => {
        //set up the utah header
        window["@utahdts/utah-design-system-header"].setUtahHeaderSettings({
          mainMenu: {
            menuItems: [
              {
                actionMenu: [
                  {
                    title: "child2-1",
                    actionMenu: [
                      {
                        title: "child2-1-1",
                        actionUrl: {
                          url: "https://www.google.com/",
                        },
                      },
                      {
                        title: "child2-1-2",
                        actionUrl: {
                          url: "https://designsystem.utah.gov/",
                        },
                      },
                      {
                        title: "child2-1-3",
                        actionUrl: {
                          url: "https://utah.gov/",
                        },
                      },
                    ],
                  },
                  {
                    title: "child2-2",
                    actionMenu: [
                     {
                        title: "child2-1-1",
                        actionUrl: {
                          url: "https://www.google.com/",
                        },
                      },
                      {
                        title: "child2-1-2",
                        actionUrl: {
                          url: "https://designsystem.utah.gov/",
                        },
                      },
                      {
                        title: "child2-1-3",
                        actionUrl: {
                          url: "https://utah.gov/",
                        },
                      },
                    ],
                  },
                ],
                title: "flyout (default)",
              },
              // inline
              {
                "childrenMenuType": 'inline',
                actionMenu: [
                  {
                    title: "child2-1",
                    actionMenu: [
                      {
                        title: "child2-1-1",
                        actionUrl: {
                          url: "https://www.google.com/",
                        },
                      },
                      {
                        title: "child2-1-2",
                        actionUrl: {
                          url: "https://designsystem.utah.gov/",
                        },
                      },
                      {
                        title: "child2-1-3",
                        actionUrl: {
                          url: "https://utah.gov/",
                        },
                      },
                    ],
                  },
                  {
                    title: "child2-2",
                    actionMenu: [
                      {
                        title: "child2-1-1",
                        actionUrl: {
                          url: "https://www.google.com/",
                        },
                      },
                      {
                        title: "child2-1-2",
                        actionUrl: {
                          url: "https://designsystem.utah.gov/",
                        },
                      },
                      {
                        title: "child2-1-3",
                        actionUrl: {
                          url: "https://utah.gov/",
                        },
                      },
                    ],
                  },
                ],
                title: "inline",
              },
              // mega
              {
                "childrenMenuType": 'mega-menu',
                actionMenu: [
                  {
                    title: "child2-1",
                    actionMenu: [
                     {
                        title: "child2-1-1",
                        actionUrl: {
                          url: "https://www.google.com/",
                        },
                      },
                      {
                        title: "child2-1-2",
                        actionUrl: {
                          url: "https://designsystem.utah.gov/",
                        },
                      },
                      {
                        title: "child2-1-3",
                        actionUrl: {
                          url: "https://utah.gov/",
                        },
                      },
                    ],
                  },
                  {
                    title: "child2-2",
                    actionMenu: [
                     {
                        title: "child2-1-1",
                        actionUrl: {
                          url: "https://www.google.com/",
                        },
                      },
                      {
                        title: "child2-1-2",
                        actionUrl: {
                          url: "https://designsystem.utah.gov/",
                        },
                      },
                      {
                        title: "child2-1-3",
                        actionUrl: {
                          url: "https://utah.gov/",
                        },
                      },
                    ],
                  },
                ],
                title: "mega-menu",
              },
            ],
            title: "Utah Design System Main Menu",
          },
          mediaSizes: {
            mobile: 640,
            tabletPortrait: 768,
            tabletLandscape: 1024,
          },
          showTitle: true,
          size: "MEDIUM",
          title: "My Utah.gov Site",
          titleURL: "/",
          utahId: false,
          footer: null,
          logo: null,
          skipLinkUrl: null,
          actionItems: null,
          onSearch: false,
        });
      });
    </script>
  </head>
  <body class="utah-design-system">
    <div>Nothing</div>
  </body>
</html>
```

**Comment by AustinHaws-Utah** [comment]
Thanks Ty for reporting this. JIRA issue added for this sprint: UDS-1273

**Comment by AustinHaws-Utah** [comment]
A fix for this has been released in version 1.7.1.

For the curious, the fix involved opening Schrödinger's box to find that the parent menu item was gobbling the click event so that the child menu wasn't triggering.
```
         __..--''``---....___   _..._    __
/// //_.-'    .-/";  `        ``<._  ``.''_ `. / // /
///_.-' _..--.'_    \                    `( ) ) // //
/ (_..-' // (< _     ;_..__               ; `' / ///
/ // // //  `-._,_)' // / ``--...____..-' /// / //
```
